### PR TITLE
feat(issue-details): Use a time range for the graph tooltip

### PR DESCRIPTION
**Pull request by leeandher**
A bit verbose, but the bar chart tooltips are calculated off of the item rather than the axis, so I needed to compute the bucketSize manually. It works though!

image

Also moves the discover button out of the graph, and into all events:

image

### CODE REVIEW

File: static/app/views/dashboards/widgetCard/chart.tsx [hunk 1]
```diff
@@ -608,7 +608,7 @@ class WidgetCardChart extends Component<WidgetCardChartProps> {
   }
 }
 
-const getBucketSize = (series: Series[] | undefined) => {
+export const getBucketSize = (series: Series[] | undefined) => {
```
Comment (scttcper) [inline static/app/views/dashboards/widgetCard/chart.tsx:611]: importing this will drag in too much stuff, i'd move it to another file or copy it out

**Comment by scttcper** [comment]
what's this fix? were the tooltips wrong?

**Comment by leeandher** [comment]
> what's this fix? were the tooltips wrong?

We only talked about it briefly but the tooltips for the bar charts should have a time range rather than a time stamp. It'll more clearly describe the bucketing we're doing.

**Comment by scttcper** [comment]
@leeandher ahhhh i'm blind perfect

**Review by scttcper — APPROVED** [review]

**Review by scttcper — APPROVED** [review]

**Pull request merged**